index.js

```diff
@@ -2150,6 +2150,56 @@ export async function createNodejsBom(path, options) {
       }
     }
   }
+  const pkgJsonLockFile = getAllFiles(path, "package-lock.json", options);
+  const pkgJsonFile = getAllFiles(path, "package.json", options);
+  const yarnLockFile = getAllFiles(path, "yarn.lock", options);
+  const pnpmLockFile = getAllFiles(path, "pnpm-lock.yaml", options);
+  if (
+    (pkgJsonLockFile?.length === 0 ||
+      yarnLockFile?.length === 0 ||
```

Comment:
Must be an and condition. The idea is we install only if there are no lock files.